nuxt.config.ts

```diff
@@ -153,15 +153,18 @@ export default defineNuxtConfig({
         quality: 75,
         screens: {
             xs: 375, // override size to match our breakpoints
-            xl: 1440, // override size to match our breakpoints
+            sm: 480, // overridden
+            vl: 1280, // overridden
+            xl: 1440, // overridden
+            xxl: 1600, // overridden
             hd: 1920, // additional size
             qhd: 2500, // additional size
         },
         // @ts-expect-error not working with [1]
         densities: '1',
         presets: {
             default: {
-                sizes: 'xs:100vw md:100vw lg:100vw vl:100vw xl:100vw hd:100vw qhd:100vw',
+                sizes: 'xs:100vw sm:100vw md:100vw lg:100vw vl:100vw xl:100vw xxl:100vw hd:100vw qhd:100vw',
```

Comment:
in the preset do we really want to have all the sizes? IMO it could be a bad move because it will add many sizes in the output. By default, Nuxt image already use all the available sizes, but in this preset we do not want all sizes.